docs/getting-started.md

```diff
@@ -264,6 +264,9 @@ ssh -i ./ubuntu-22.04.id_rsa root@172.16.0.2
 
 # Use `root` for both the login and password.
 # Run `reboot` to exit.
+#
+# For internet access in the guest, run `ip route add default via 172.16.0.1 dev eth0`
+# For DNS resolution in the guest, run `echo "nameserver 8.8.8.8" > /etc/resolv.conf`
```

Comment:
I think since these are required for other parts to work it makes sense to have them by default.

```suggestion

# Setup internet access in the guest
ip route add default via 172.16.0.1 dev eth0

# Setup DNS resolution in the guest
echo "nameserver 8.8.8.8" > /etc/resolv.conf
```

Reply:
What other parts? The guide ends here.

Reply:
> What other parts? The guide ends here.

The other parts above of the guide that begin setting up the internet connection.

Reply:
That makes sense, however, the script should run inside guest, so it would be:

```
# Setup internet access in the guest
ssh -i ./ubuntu-22.04.id_rsa root@172.16.0.2  "ip route add default via 172.16.0.1 dev eth0"

# Setup DNS resolution in the guest
ssh -i ./ubuntu-22.04.id_rsa root@172.16.0.2  'echo "nameserver 8.8.8.8" > /etc/resolv.conf'
```

Reply:
On introducing commands via ssh, it become important to wait long enough for them to succeeed, that means the `sleep 0.015s` is too fast for some systems. I run this on 6CPU 16GB machine, and yet it never runs fast enough to ssh. 

Hence, if we're to include ssh command I'd also suggest increasing sleep to a second or two.

Reply:
I'm happy to do both changes - if above sounds reasonable?

Reply:
Thank you all for first review. Just checking in to see if there's feedback on this that I can address?  

My follow up question was regarding suggested change - if that's not applicable or incorrect for this documentation, please feel free to commit the suggested change.

Reply:
If the ssh commands can work I think there is no problem increasing the sleep time, I think this is a good approach.

Reply:
In my tests 2s works every time and hence I updated PR as such before running ssh commands.

This is much better than my original PR - thank you for the reviews

Reply:
lgtm